### ISSUE TITLE
how to programmatically display loader in case of single select 

### ISSUE THREAD

**Issue by yash-habilelabs** [riophae/vue-treeselect#282]
in my last question #275 discussed about infinite scroll, but now my problem is as per doc by setting loading= true we can display loader, but when i tried apply this i am unable.
in short want to display loader like more are loading.

**Comment by riophae** [comment]
I'm sorry, that was my fault. The `loading` prop was not being handled. Now I decide to remove the `loading` prop since I think it's better to show a spinner inside the menu.

Now with the newly published v0.3.0, you can achieve that by:

```vue
<template>
  <treeselect ...>
    <treeselect-tip slot="after-list" v-if="isFetchingOptions" type="loading" icon="loader">Loading...</treeselect-tip>
  </treeselect>
</template>

<script>
  import TreeselectTip from '@riophae/vue-treeselect/src/components/Tip.vue'

  export default {
    components: {
      TreeselectTip,
    },
  }
</script>
```

**Comment by yash-habilelabs** [comment]
It's working thanks @riophae . Now i think we can try the other way for the infinite-scroll by this slot.
